html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Forms</title>
+  </head>
+  <body>
+
+    <h1>Coding Form!</h1>
+
+    <div>
+      <label for="email">Email</label>
+    </div>
+
+    <input type="text" name="email" />
```

Comment:
In order to attach an input to a label, the `<input>` element should have an id attribute that matches the value of the "for" attribute. So in this case, your input needs an `id="email"` in order to attach it to your label